develop/getting-started/migrate-mappings.md

```diff
@@ -0,0 +1,45 @@
+# Updating Yarn mappings in a Java codebase
+
+Loom allows semi-automatic updating of the mappings used in a Java codebase. Due to frequent changes in Yarn, this can be a useful tool for keeping a codebase up-to-date with the newest changes.
```

Comment:
Yarn no longer has frequent changes, most of the time these days I wouldnt reccomend using migrate mappings.

Reply:
it's useful for when switching from yarn to mojmaps